src/test/scala/performanceanalysis/ParsedLineSpec.scala

```diff
@@ -0,0 +1,62 @@
+package performanceanalysis
+
+import java.time.LocalDateTime
+
+import performanceanalysis.base.SpecBase
+
+class ParsedLineSpec extends SpecBase {
+
+  "ParsedLine" must {
+
+    "Extract a date" in {
+      val line = """[INFO] [04/19/2016 14:17:16.829] [Some.ClassName] Some action took 101 ms"""
+      val parser = LineParser("")
+      val result = parser.parse(line)
+      result.dateTime.get shouldBe LocalDateTime.parse("2016-04-19T14:17:16.829")
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://www.codacy.com/app/scala-academy/performance-analysis/file/2586410711/issues/source?bid=3220915&fileBranchId=3304352#l15)